css/style.css

```diff
@@ -1,3 +1,7 @@
+body {
+  overflow: hidden;
```

Comment:
a.k.a. let’s hide everything under the carpet. :-)

Please gimme a few minutes to see if I can propose an alternative.